gateway/proxy.go

```diff
@@ -35,6 +36,12 @@ var defaultTimeoutValues map[string]Upstream = map[string]Upstream{
 		HTTPClientTLSHandshakeTimeout:   time.Second * 5,
 		HTTPClientResponseHeaderTimeout: time.Second * 5,
 	},
+	RULER: {
+		HTTPClientTimeout:               time.Second * 10,
+		HTTPClientDialerTimeout:         time.Second * 3,
+		HTTPClientTLSHandshakeTimeout:   time.Second * 4,
+		HTTPClientResponseHeaderTimeout: time.Second * 3,
```

Comment:
```suggestion
		HTTPClientTimeout:               time.Second * 15,
		HTTPClientDialerTimeout:         time.Second * 5,
		HTTPClientTLSHandshakeTimeout:   time.Second * 5,
		HTTPClientResponseHeaderTimeout: time.Second * 5,
```
No reason, for now, to have a different timeout from the others.

Reply:
when we talked on slack, we decided to have 10 seconds for the ruler's HTTPClientTimeout.

Reply:
did we?. I can't remember now 😅 . Can you remind me a reason for that?